README.md

```diff
@@ -1,20 +1,29 @@
 # docs.daml.com
 
-This repo manages the full build of technical documentation into https://docs.daml.com from the following repos:
+This repo manages the [docs.daml.com](https://docs.daml.com) website.
 
-* [Daml docs](https://github.com/digital-asset/daml/tree/main/docs)
-* [Canton docs](https://github.com/DACH-NY/canton/tree/main/docs)
-* [Daml-Finance docs](https://github.com/digital-asset/daml-finance/tree/main/docs)
+In the `docs` folder you will find a folder for each version that is still
+supported by the docs team. Making changes to versions that are not in the docs
+folder is not supported.
 
-:bulb: Docs in this repo build the main TOC with pointers to pages from the above repos.
+Whenever a PR is merged into this repo, the docs website is brought up-to-date
+with the repo. This means that:
 
-The process for updating the docs TOCs has some potential pitfalls. Follow the instructions in this README *carefully* and in order.
+- Every version in the `docs` folder that has changed compared to what is
+  currently on the website is rebuilt and pushed to the website.
+- The root of the website is updated, if needed, to match the contents of the
```

Comment:
Could we explain clearly what is the "root of the website"?

Reply:
`https://docs.daml.com`, as opposed to a version-specific subsite like `https://docs.daml.com/2.6.0`.